src/sections/Section2/Section2.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Container, Row, Col } from "reactstrap";
+import content from "../../static/content/content";
+import Section2Background from "../../sections/Section2/Section2Background";
+import ContentBox from "../../components/ContentBox/ContentBox";
+import FieldBox from "../../components/FieldBox/FieldBox";
+
+const Section2 = () => {
+  const sectionHeader = content[1].header;
+  const sectionContent = content[1].resume;
+  const sectionId = content[1].id;
+  const fieldContent = content[1].formText;
+
+  return (
+    <div className="Section2" id="section2">
+      <Section2Background />
```

Comment:
condense code, see about making something reusable since this is similar to section 1